pkg/config/config.go

```diff
@@ -86,7 +86,7 @@ func LoadIPAMConfig(bytes []byte, envArgs string, extraConfigPaths ...string) (*
 			RangeEnd:   n.IPAM.RangeEnd,
 		}
 
-		n.IPAM.IPRanges = append(n.IPAM.IPRanges, oldRange)
```

Comment:
I don't think you actually deleted the "oldRange" from the configuration (below, lines 121 to 129). 

Honestly I'm unsure if we want to do that. Let's ask @dougbtv 's opinion.

Reply:
I deleted "oldRange" completely right after copying it as the first element of `IPRanges` and updated the corresponding tests too :)